vms/event/services.py

```diff
@@ -1,18 +1,30 @@
 from django.core.exceptions import ObjectDoesNotExist
 from event.models import Event
 
+#check if the event exists and is not empty
+def event_not_empty(event_id):
+    result = True
+    event = get_event_by_id(event_id)
+    if not event:
+        result = False
+    return result
+
 #need to check that this event is not accociated with any jobs, 
```

Comment:
@vubo Just an FYI, I prefer one line docstrings if the function is straightforward/easily understandable. If it is a complicated function with lots of passed parameters then please use a multi-line docstring. In general, I prefer self documenting code -- less comments and better names :)